ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/advanced-gas-fee-inputs-priorityfee.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import { PRIORITY_LEVELS } from '../../../../../shared/constants/gas';
+import { SECONDARY } from '../../../../helpers/constants/common';
+import { decGWEIToHexWEI } from '../../../../helpers/utils/conversions.util';
+import { getAdvancedGasFeeValues } from '../../../../selectors';
+import { useCurrencyDisplay } from '../../../../hooks/useCurrencyDisplay';
+import { useGasFeeContext } from '../../../../contexts/gasFee';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { useUserPreferencedCurrency } from '../../../../hooks/useUserPreferencedCurrency';
+import FormField from '../../../ui/form-field';
+
+import AdvancedGasFeeInputSubtext from '../advanced-gas-fee-input-subtext';
+
+const AdvancedGasFeeInputPriorityFee = () => {
```

Comment:
What do you think about renaming this to MaxPriorityFeeInput?

Reply:
👍 changed